data/json/itemgroups/guns.json

```diff
@@ -309,6 +310,15 @@
     "id": "guns_launcher_rocket",
     "//": "Factory manufactured rocket launchers (excludes improvised weapons)",
     "items": [
+      [ "LAW_Packed", 200 ],
+      [ "RPG", 5 ],
+      [ "RPG-7_ammo", 10 ],
```

Comment:
This wasn't added here initially since it's weird to use russian weapon in the US by US army.